integration-tests/capacity/staking.test.ts

```diff
@@ -13,12 +13,17 @@ import {
 import { firstValueFrom } from "rxjs";
 
 describe("Capacity Staking Tests", function () {
-    const accountBalance: bigint = 200n * DOLLARS;
+    const accountBalance: bigint = 2n * DOLLARS;
```

Comment:
The account was overfunded which isn't necessarily a problem. I reduced some initial account funding values to reduce the amount of token transferred from funding source on Rococo